framework/src/modules/interoperability/cc_commands/channel_terminated.ts

```diff
@@ -15,14 +15,20 @@
 import { BaseCCCommand } from '../base_cross_chain_command';
 import { CROSS_CHAIN_COMMAND_ID_CHANNEL_TERMINATED } from '../constants';
 import { channelTerminatedCCMParamsSchema } from '../schema';
+import { CCCommandExecuteContext } from '../types';
 
 export class CCChannelTerminatedCommand extends BaseCCCommand {
 	public ID = CROSS_CHAIN_COMMAND_ID_CHANNEL_TERMINATED;
 	public name = 'channelTerminated';
 	public schema = channelTerminatedCCMParamsSchema;
-	// TODO
+
+	public async execute(context: CCCommandExecuteContext): Promise<void> {
+		await this._createTerminatedStateAccount(context.ccm.sendingChainID);
+	}
```

Comment:
I missed it out but we need to call `interoperabilityStore.createTerminatedStateAccount` and the `interoperabilityStore` we can add in `CCCommandExecuteContext`

Reply:
Apparently adding/calling `interoperabilityStore` either in `context` or `BaseCommand` won't work because both are at higher level and introduce cyclic dependency.